Core/Core/Inbox/ComposeMessage/Model/ComposeMessageInteractorLive.swift

```diff
@@ -50,7 +148,31 @@ public class ComposeMessageInteractorLive: ComposeMessageInteractor {
         }
     }
 
-    public func deleteFile(file: File) -> AnyPublisher<Void, Never> {
+    // MARK: Private helpers
+
+    private func setupAttachmentListBinding() {
+        alreadyUploadedFiles.setFailureType(to: Error.self)
+            .combineLatest(with: files)
+        .receive(on: DispatchQueue.main)
```

Comment:
You might want to inject a `AnySchedulerOf<DispatchQueue>` instance so it's easier to unit test.